tests/server/routes/get-verify-email.js

```diff
@@ -97,26 +90,30 @@ define([
         url: '/verify_email'
       };
 
-      mockModule(mocks).process(req, res, () => {
+      res.redirect = () => {
         assert.equal(logger.error.callCount, 0);
         assert.equal(ravenMock.ravenMiddleware.captureMessage.callCount, 0);
         assert.equal(ravenMock.ravenMiddleware.captureError.callCount, 0);
 
-        req.query.something = 'else';
-
-        mockModule(mocks).process(req, res, () => {
+        res.redirect = () => {
+          // calling with `req.query.something` captures a message to Sentry
           assert.equal(logger.error.callCount, 0);
-          assert.equal(ravenMock.ravenMiddleware.captureMessage.callCount, 0);
+          assert.equal(ravenMock.ravenMiddleware.captureMessage.callCount, 1);
```

Comment:
I'm a little confused here, test says `no logs` but it checks that it logged.

Reply:
updated description, good find!